src/test/java/com/iota/iri/TransactionTestUtils.java

```diff
@@ -138,9 +157,9 @@ public static TransactionViewModel createTransactionWithTrunkAndBranch(String tr
      * @param branch The branch transaction hash
      * @return The transaction trits
      */
-    public static byte[] getTransactionWithTrunkAndBranch(Hash trunk, Hash branch) {
-        byte[] trits = new byte[TransactionViewModel.TRINARY_SIZE];
-        return getTransactionTritsWithTrunkAndBranch(trits, trunk, branch);
+    public static byte[] getRandomTransactionWithTrunkAndBranch(Hash trunk, Hash branch) {
```

Comment:
Maybe worth specifying that this method, in contrast to the `create*` methods, do not calculate Tx's hash, it will be random / not correct instead. Is this intended and accounted for?
Also: since we are changing the naming: from its name, I would expect that this method returns a transaction object, it returns bytes instead. Same for `getTransactionWithTrunkAndBranchTrits`.